docs/source/solving-interface.rst

```diff
@@ -837,8 +837,68 @@ in the transpose nullspace by providing a
 :class:`~firedrake.nullspace.VectorSpaceBasis` with the
 ``transpose_nullspace`` keyword argument to :func:`~.solve`.
 
-Singular operators in mixed spaces
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Achieving exact mean zero on nonuniform meshes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Trying the above example on a nonuniform mesh with variable cell areas
+will give a solution that does not have zero spatial mean. This is
```

Comment:
Mention that it does give a solution that has satisfies $\int u dx = c$ for some $c = O(h^2)$ (i.e. it converges to zero at second order)?